### PR TITLE
Add BAU expectations

### PR DESCRIPTION
**Pull request by chrisblackburn**
A list of things an application and platform must provided to be considered ready for our Support Team to run

### CODE REVIEW

File: team-norms/bau_expectations.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Business As Usual (BAU) Expectations
+
+Teams must deliver software systems in accordance with our Business As Usual Expectations that make software systems easier to run, debug, and improve.
+
+The Made Tech Support Team requirements are identified below. Where the application is transitioning to a customer support team, you should liaise with the customer team to understand their specific needs.
+
+## Application Requirements
+
+- Centralised Logging (Papertrail)
+- Application Performance Monitoring (Datadog)
```
Comment (davidwinter) [inline team-norms/bau_expectations.md]: Unsure if we can fully go with this _currently_. I'm pretty sure when we started using Datadog for infra, the APM was free. Now they're charging for it too separate.

File: team-norms/bau_expectations.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Exception Tracking (Sentry)
+- Automated Dependency Upgrades (Dependabot)
+- Automated Security Patches (Snyk)
```
Comment (davidwinter) [inline team-norms/bau_expectations.md]: Possibility we might be able to use Github security alerts for this now, which wouldn't require an additional charge. Needs testing though how good it actually is, if at all.

Comment (chrisblackburn) [reply team-norms/bau_expectations.md]: OK, what do you want to do in here? Leave blank to force conversation with team?

File: team-norms/bau_expectations.md (hunk 1 above)
Comment (craigjbass) [reply team-norms/bau_expectations.md]: Yeah agree - we should get APM into projects as standard but all the options are fairly pricy.

File: team-norms/bau_expectations.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Teams must deliver software systems in accordance with our Business As Usual Expectations that make software systems easier to run, debug, and improve.
+
+The Made Tech Support Team requirements are identified below. Where the application is transitioning to a customer support team, you should liaise with the customer team to understand their specific needs.
+
+## Application Requirements
+
+- Centralised Logging (Papertrail)
+- Application Performance Monitoring (Datadog)
+- Exception Tracking (Sentry)
+- Automated Dependency Upgrades (Dependabot)
```
Comment (craigjbass) [inline team-norms/bau_expectations.md]: We're using package manager installed gitlab (k8s helm chart). 

Downside is that dependency upgrades don't work with gitlab. 

Upside is we have a make task that upgrades dependencies in < 30 seconds and tests that are safe enough to rely upon.

Is Dependabot a requirement?

File: team-norms/bau_expectations.md (hunk 1 above)
Comment (chrisblackburn) [reply team-norms/bau_expectations.md]: @davidwinter did we figure out New Relic was going to do the job we need on the free tier for now?

File: team-norms/bau_expectations.md (hunk 3 above)
Comment (chrisblackburn) [reply team-norms/bau_expectations.md]: I'd say everything here is a requirement unless you can reach agreement with the support team that they're going to be able to support your own custom setup

**Comment by lukemorton** [comment]
Stale?

**Pull request closed without merging**